core/templates/dev/head/pages/exploration-editor-page/services/exploration-rights.service.ts

```diff
@@ -136,7 +136,7 @@ angular.module('oppia').factory('ExplorationRightsService', [
           '/createhandler/moderatorrights/' +
           ExplorationDataService.explorationId);
 
-        $http.put(explorationModeratorRightsUrl, {
+        return $http.put(explorationModeratorRightsUrl, {
```

Comment:
If you aren't using data above, is this change needed?

Reply:
I changed it just to follow the same "pattern" as the other methods. However, this is not a important change, so I removed it. Done.